lib/ohai/plugins/solaris2/network.rb

```diff
@@ -101,7 +101,8 @@ def full_interface_name(iface, part_name, index)
     cint = nil
 
     so.stdout.lines do |line|
-      if line =~ /^([0-9a-zA-Z\.\:\-]+)\S/
+      # if line =~ /^([0-9a-zA-Z\.\:\-]+)\S/
+      if line =~ /^([0-9a-zA-Z\.\:\-]+): \S+ mtu (\d+) index (\d+)/
```

Comment:
Like we discussed - need to update this regex to allow for but not require the last :